### DISCUSSION TITLE
How do we deploy to multiple environment using docker-compose?

### DISCUSSION THREAD

**Discussion by indyarocks** [basecamp/kamal#490] [Q&A]
I want to have the old fashioned way of deploying the docker containers with docker-compose to staging and production independently like Capistrano flexibility. 

Docker-compose enables me to build image on the fly, instead of having two steps of building image - pushing it to container registry and then pull it to spin the container.

**Comment by pythoninthegrass** [comment]
Same here @indyarocks . It would be nice to be able to choose either a dockerfile or compose file. Without the latter I'll have to duplicate work/yaml and learn kamal's directives vs industry standards.

@dhh is compose supported and just not documented?

**Comment by dhh** [comment]
Kamal’s primary focus is on setups spanning multiple machines now or in the near future. Compose is not suitable for that. No current plans to support compose.

**Reply by pythoninthegrass** [reply]
Appreciate the swift response! Really like Kamal and will see if it's the right fit regardless 🤞
